apps/api/src/app/controllers/command/command.controller.integration.spec.ts

```diff
@@ -0,0 +1,113 @@
+import { CommandHandlerService, FluxStandardAction } from '@coscrad/commands';
+import { INestApplication } from '@nestjs/common';
+import * as request from 'supertest';
+import { AddSong } from '../../../domain/models/song/commands/add-song.command';
+import { AddSongHandler } from '../../../domain/models/song/commands/add-song.command-handler';
+import buildInMemorySnapshot from '../../../domain/utilities/buildInMemorySnapshot';
+import generateRandomTestDatabaseName from '../../../persistence/repositories/__tests__/generateRandomTestDatabaseName';
+import TestRepositoryProvider from '../../../persistence/repositories/__tests__/TestRepositoryProvider';
+import { DTO } from '../../../types/DTO';
+import httpStatusCodes from '../../constants/httpStatusCodes';
+import setUpIntegrationTest from '../__tests__/setUpIntegrationTest';
+
+export const dummyUuid = '9b1deb4d-3b7d-4bad-9bdd-2b0d7b3dcb6d';
+
+const validCommandFSA: FluxStandardAction<DTO<AddSong>> = {
+    type: 'ADD_SONG',
+    payload: {
+        id: dummyUuid,
+        title: 'test-song-name (language)',
+        titleEnglish: 'test-song-name (English)',
+        contributorAndRoles: [],
+        lyrics: 'la la la',
+        audioURL: 'https://www.mysound.org/song.mp3',
+        lengthMilliseconds: 15340,
+    },
+};
+
+const validPayload = validCommandFSA.payload;
+describe('AddSong', () => {
```

Comment:
Add comment stating that this is a higher level test that just asserts that the controller is correctly mapping the various exceptions and results  to the appropriate response codes.